openupgrade_scripts/scripts/event/17.0.1.8/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,67 @@
+---Models in module 'event'---
+---Fields in module 'event'---
+event        / event.event              / _order                        : _order is now 'date_begin, id' ('date_begin')
+event        / event.event              / activity_user_id (many2one)   : not related anymore
+event        / event.event              / activity_user_id (many2one)   : now a function
+# NOTHING TO DO
+event        / event.event              / auto_confirm (boolean)        : DEL
+# NOTHING TO DO: Attendees are now created confirmed (state=open)
+event        / event.event              / badge_format (selection)      : NEW required, selection_keys: ['A4_french_fold', 'A6', 'four_per_sheet'], hasdefault: default
+event        / event.event              / badge_image (binary)          : NEW attachment: True
+# NOTHING TO DO
+event        / event.event              / lang (selection)              : NEW selection_keys: function
+# NOTHING TO DO: It is not required, it can be empty, we do not set the value of the old records
+event        / event.event              / message_main_attachment_id (many2one): DEL relation: ir.attachment
+event        / event.event              / rating_ids (one2many)         : NEW relation: rating.rating
+event        / event.event              / registration_properties_definition (properties_definition): NEW
+# NOTHING TO DO
+event        / event.event.ticket       / _order                        : _order is now 'event_id, sequence, name, id' ('id')
+event        / event.event.ticket       / color (char)                  : NEW hasdefault: default
```

Comment:
How was the colors handled previously? All the tickets with the same color are OK?